app/models/host_report.rb

```diff
@@ -26,13 +25,15 @@ class HostReport < ApplicationRecord
   }.freeze
 
   scoped_search relation: :host, on: :name, complete_value: true, rename: :host, aliases: %i[host_name]
+  scoped_search relation: :proxy, on: :name, complete_value: true, rename: :proxy
   scoped_search relation: :organization, on: :name, complete_value: true, rename: :organization
   scoped_search relation: :location, on: :name, complete_value: true, rename: :location
   scoped_search on: :reported_at, complete_value: true, default_order: :desc, rename: :reported, only_explicit: true, aliases: %i[last_report reported_at]
-  scoped_search on: :host_id, complete_value: false, only_explicit: true
-  scoped_search on: :proxy_id, complete_value: false, only_explicit: true
   scoped_search on: :format, complete_value: { plain: 0, puppet: 1, ansible: 2 }
-  scoped_search relation: :report_keywords, on: :name, complete_value: true, rename: :keyword
+  # This is to simulate has_many :report_keywords relation for scoped_search library to work
+  # NOTE: This won't work for any other purpose
+  reflections['report_keywords'] = ReportKeyword
```

Comment:
This is the only simple way I found to make autocompletion work otherwise we would need something like https://github.com/marshall-lee/has_array_of

Reply:
I am fine with this if it works and solves our problem.